### DISCUSSION TITLE
Status

### DISCUSSION THREAD

**Discussion by jrbarnes9** [ONSdigital/design-system#77] [Feedback]
Use this place to discuss the [status component](https://ons-design-system.netlify.com/components/status) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by glancyons** [comment]
Should this have `role="status"` added to meet WCAG:
https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=413#status-messages
Assuming Situation A
